utils/dos2unix/patches/010-deactivate-nls.patch

```diff
@@ -0,0 +1,10 @@
+--- a/Makefile
++++ b/Makefile
+@@ -50,7 +50,6 @@ MAC2UNIX        = mac2unix
+ UNIX2MAC        = unix2mac
+ 
+ # Native Language Support (NLS)
+-ENABLE_NLS      = 1
```

Comment:
Could this be replaced by `MAKE_FLAGS+= ENABLE_NLS=`? So we won't need to keep a patch here ;)
Thank you!

Reply:
Thanks for the hint, yes that works too.  I updated the pull request.